ext/page_log/palite/palite.cpp

```diff
@@ -147,15 +147,51 @@ hexdump(const void *data, size_t size)
 
     return buffer;
 }
-} // namespace
+} /* namespace */
 
 static const char *
-palite_verbose_item(const WT_ITEM *buf)
+verbose_item(const WT_ITEM *buf)
```

Comment:
np: 
```suggestion
item_hexdump(const WT_ITEM *buf)
```

Or even better, can we specify std::formatter specialisation to automatically do that?

Reply:
We already have several `WT_ITEM`-related functions. So I left it as is to keep it consistent.